test/functional/fixtures/api/es-next/browser-info/testcafe-fixtures/browser-info-test.js

```diff
@@ -0,0 +1,45 @@
+fixture `Browser information`;
+
+
+// NOTE: the possible values based on our CI functional test configs:
+const testedBrowserNames = [
+    'firefox',
+    'chrome',
+    'safari',
+    'internet explorer'
+];
+const testedPlatforms    = [
+    'desktop',
+    'mobile',
+    'tablet'
+];
+const testedOs           = [
+    'windows',
+    'macos',
+    'ios',
+    'linux'
+];
+
+function isNotValidValue (value, expectedValues) {
+    const loweredValue = value.toLowerCase();
+
+    expectedValues.forEach(function (expectedValue) {
+        if (loweredValue === expectedValue)
+            return false;
+
+        return true;
+    });
+}
+
+test
```

Comment:
At present, you've checked that current browser information value is one of expected values.
I prefer an another testing plan: 
* add server tests for all cases: empty user agent, valid user agent. check `headless` calculation.
* run an empty page only in chrome, retrieve the user agent, parse it with bowser and compare with `t.bowser`.